.idea/codeStyleSettings.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I believe you should not commit any .idea/* files to this repository. Remove them please